File: src/main/scala/li/cil/oc/integration/tmechworks/ModTMechworks.scala
package li.cil.oc.integration.tmechworks

import li.cil.oc.api.Driver
import li.cil.oc.integration.ModProxy
import li.cil.oc.integration.Mods

object ModTMechworks extends ModProxy {
  override def getMod = Mods.TMechWorks

  override def initialize() {
    Driver.add(new DriverDrawBridge)
  }
}

File: src/main/scala/li/cil/oc/integration/tmechworks/ModTMechworks.java
package li.cil.oc.integration.tmechworks;

import li.cil.oc.api.Driver;
import li.cil.oc.integration.IMod;
import li.cil.oc.util.mods.Mods;

public final class ModTMechworks implements IMod {
    @Override
    public Mods.Mod getMod() {
        return Mods.TMechWorks();
    }

    @Override
    public void initialize() {
        Driver.add(new DriverDrawBridge());
    }
}
